docs/certificate_check.adoc

```diff
@@ -275,7 +275,7 @@ Content-Type: application/ocsp-response
 
 Eine Beispielhafte OCSP-Response ist unter folgendem Link zu finden:
 
-link:../resources/examples/ti-dienste/cert/response_ocspResponse.der[OCSP Response]
+link:https://raw.githubusercontent.com/gematik/eRezept-Examples/main/API-Examples/2023-07-01/certificate_check/04_response_ocspResponse.der[OCSP Response]
```

Comment:
Das ist ja ein Link in der richtigen Ansicht. Sollte die Datei nicht bleiben?

Reply:
Ich bin der Meinung, nein. Es passt zu den Beispielen, selbst wenn es ein Link ist und nicht eingebettet wie die anderen.

Reply:
Sollte der Link direkt zum Download der Datei führen oder hier hin: https://github.com/gematik/eRezept-Examples/blob/main/API-Examples/2023-07-01/certificate_check/04_response_ocspResponse.der ?

Was würde man als Entwickler erwarten/brauchen?

Reply:
Stimmt, das war schon so, aber ich schaue, wie wir das ändern können.

Reply:
Das geht direkt zum Download der Datei.
https://github.com/gematik/api-erp/blob/unbundle-examples/docs/certificate_check.adoc

Reply:
Genau,ist das gewollt?

Reply:
Ich werde sagen, dass das gewollt ist. Man kann die Datei nur lokal ansehen, nicht auf GitHub.